### ISSUE TITLE
Deserialize empty string error message

### ISSUE THREAD

**Issue by prasmussen** [MidasLamb/non-empty-string#5]
Thanks for this package, but I got a confusing error when trying to deserialize an empty string:
```
invalid value: string \"\", expected an integer between -2^31 and 2^31
```

Looks like a copy-paste oversight:
https://github.com/MidasLamb/non-empty-string/blob/86b700166973f433332688d9628f9b7e8613bb7b/src/serde_support.rs#L38

**Comment by MidasLamb** [comment]
You're right, I'll get this fixed ASAP!

**Comment by MidasLamb** [comment]
Fixed by #6, published as [v0.2.1]. 

Error message will now be:
```
invalid value: string \"\", expected a string with a length of more than 0
```

If you think the message could be improved still, let me know!

[v0.2.1]: https://crates.io/crates/non-empty-string/0.2.1
